src/constants.js

```diff
@@ -25,6 +25,26 @@ export const LINE_NAMES = [
   'Somatic'
 ];
 
+// Name for all the lines
+export const DESIGNERS_NAMES = [
+  'Arushi Bandi, Maya Pandagurun',
```

Comment:
Obviously a placeholder for now, but on top of consolidating the data structure for all the line info, we should also make the names be in an array, so that way we don't have to hard code in the commas.

i.e.

```
[
  'Arushi Bandi',
  'Maya Pandagurun'
]
```